### ISSUE TITLE
How to get picture cache from disk by url? 

### ISSUE THREAD

**Issue by yumi0629** [bumptech/glide#1468]
I display a picture from network, so it will be cached on disk. But how can I get picture cache from disk by url? As `ImageLoader` provides a method "`getDiskCache().get(url)`" to get appointed local cache. Does Glide has the similar method to support it?

**Comment by TWiStErRob** [comment]
Yes, if the original images was loaded with `.diskCacheStrategy(SOURCE or ALL)`, then you can access the file wih `.downloadOnly()`.

Depending on why you need it, there are several ways to go from here. You may not even need `downloadOnly()`, but `.asBitmap().toBytes()`.
